using-timescaledb/continuous-aggregates.md

```diff
@@ -0,0 +1,311 @@
+The various options used to create the continuous aggregate view, as well as its
+definition can be found in the [`timescaledb_information.continuous_aggregates`
+view][continuous-aggregates-info], and information about the state and progress
+of the materialization background worker jobs can be found in the
+[`timescaledb_information.continuous_aggregate_stats`
+view][continuous-aggregate-stats]. These views can be quite useful for
+administering continuous aggregates and tuning other options noted below. 
+
+---
+
+### Altering and Dropping Continuous Views [](alter)
+In addtion to these important options, there are several other options that may
+be set or modified after the view has been created. These may be set by running
+an `ALTER VIEW` command like so: 
+```sql
+ALTER VIEW device_summary SET (timescaledb.refresh_interval = '10 min');
+```
+Which sets the interval at which the view refreshes to 10 minutes.
+
+
+Other alterations to the continuous aggregate view are currently disallowed. To
+alter a continuous aggregate view in other ways it must be dropped and
+re-created; this can entail some time to re-calculate aggregations. 
+
+A continuous aggregate may be dropped by using the `DROP VIEW` command. 
+```sql
+DROP VIEW device_summary;
+```
+
+---
+
+
+### Dropping Data with Continuous Aggregates Enabled [](dropping-data)
+When dropping data in a raw hypertable using the [`drop_chunks`
+function][api-drop-chunks] that has a continuous aggregate created on it, we
+must specify the `cascade_to_materializations` argument to the `drop_chunks`
+call. Currently, the only option for this argument is `true`, which will cause
+the continuous aggregate to drop all data associated with any chunks dropped from the
+raw hypertable. Further data retention options are planned for future releases
+(see [future work](#future-work)). 
+
+The same argument must also be supplied to the [`add_drop_chunks_policy`
+function][api-add-drop-chunks] when creating a data retention policy for a
+hypertable with a continuous aggregate.  
+
+---
+
+### Best Practices [](best-practices)
+
+**Modifying the Materialization Hypertable**
+Advanced users may find the need to modify certain properties of the
+materialization hypertable (ie chunk size etc.) or to create further indexes,
+we can find the name of the materialization hypertable in the
+`timescaledb_information.continuous_aggregates` view ([API Docs][continuous-aggregates-info]).
+
+And then modify the materialization hypertable as if it were a normal
+hypertable. For instance, we may want to set the materialization hypertable's
+`chunk_time_interval` to something other than the default, this can be
+accomplished by running [`set_chunk_time_interval`][api-set-chunk-interval] on
+the materialization hypertable.  
+
+**Indexes on the Materialization Hypertable** 
+By default indexes on each group by column + the `time_bucket` column (i.e.
+indexes on {foo, bucket}, {bar, bucket}...) will be created (setting
```

Comment:
could you use the columns from the view example (device_summary). Or give an example here and use those columns instead of "foo" and "bar".